src/AppBundle/Service/FacebookInfoService.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: hamler
+ * Author: Viktoras Bezubec
+ * Date: 17.11.22
+ * Time: 11.50
+ */
+
+namespace AppBundle\Service;
+
+use Facebook\Exceptions\FacebookResponseException;
+use Facebook\Exceptions\FacebookSDKException;
+use Facebook\Facebook;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\Session\Session;
+
+class FacebookInfoService
+{
+    private $facebook;
+    private $accessToken;
+
+    /**
+     * FacebookInfoService constructor.
+     *
+     * @param ContainerInterface $container
+     * @param Session $session
+     */
+    public function __construct(ContainerInterface $container, Session $session)
```

Comment:
It's bad to inject whole container, because it's huge... inject only needed parameters or needed services...